client/x/epochs/keeper/keeper.go

```diff
@@ -0,0 +1,53 @@
+package keeper
+
+import (
+	"cosmossdk.io/collections"
+	"cosmossdk.io/core/event"
+	"cosmossdk.io/core/store"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	"github.com/piplabs/story/client/x/epochs/types"
+)
+
+type Keeper struct {
+	cdc   codec.BinaryCodec
+	hooks types.EpochHooks
+
+	// NOTE(Narangde): Add storeService and EventService manually instead of Environment
+	storeService store.KVStoreService
```

Comment:
There is no specific reason. `storeService` starts in lower case as done in other module, and used upper case for `EventService` as done in [cosmos-sdk](https://github.com/cosmos/cosmos-sdk/blob/aa8bf41eb94d073c834718cda24b6be23943a52a/core/appmodule/v2/environment.go#L20).

I wanted to make diff as minimal as possible comparing with cosmos-sdk, so I use the cosmos-sdk way.
When cosmos-sdk v0.52.x is released, this module can be replaced to that of cosmos-sdk. Before that, we need to upstream the cosmos-sdk. In that case, minimal diff would be good to us. 

I tested the epochs module in local env, but have not checked if the event service works. But it will work since there is common way in cosmos-sdk as you said. I will test it.

Reply:
`storeService` shouldn't be accessed publicly, so it's lowercased.